scripts/make-serializer-pickle-files

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+cat << "EOF" > serializer-pickle-files.lock
+# This file was automatically generated by running `make serializer-pickle-files.lock`.
+# The purpose of this file is to prevent people from innocently adding new usages
+# of serializer='pickle' in celery @task declarations, since such usage is considered deprecated in this project.
+# As we remove usages, this file will be updated to reflect that and prevent unintended backsliding.
+
+EOF
+
+cat <(grep -R "serializer='pickle'" --include='*.py' . | cut -d':' -f1 | cut -c3- | sort | uniq -c) >> serializer-pickle-files.lock
```

Comment:
nit: `grep` can count for you, albeit doing so requires some different output massaging. This could work if you're interested:

```bash
grep -REc "serializer=['\"]pickle['\"]" --include='*.py' . \
    | grep -Ev ':0$' \
    | awk -F: '{sub(/^\.\//, "", $1); printf("%3i %s\n", $2, $1)}' \
    >> serializer-pickle-files.lock
```
Where:
- `grep -E ...` == extended regex
- `"serializer=['\"]pickle['\"]"` matches both single and double quotes
- `grep -Ev ':0$'` drops the "zero count" results
- `awk -F: ...` reformats the output (e.g. `./corehq/apps/users/tasks.py:11` --> ` 11 corehq/apps/users/tasks.py`)
- the `sub(/^\.\//, "", $1); ` command is only needed to strip the leading `./` off of the first field (`$1`).

Reply:
the no trailing dot thing in my experience is platform-dependent. On my local machine (Mac) for example removing the `.` didn't change the output (still started with `./`). Cut the way I'm using it here works the same everywhere so  I decided to go with that.

Reply:
Thanks I used a variation of your awk suggestion to normalize the whitespace. Hopefully that'll do it :)